README.md

```diff
@@ -47,3 +48,14 @@ This project is licensed under the [Apache License Version 2.0](LICENSE).
 ### Contribution
 
 Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in this project by you shall be licensed as Apache License Version 2.0 without any additional terms or conditions.
+
+[version-badge]: https://img.shields.io/crates/v/veecle-os?style=flat-square&logo=image%2Fpng%3Bbase64%2CiVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAIGNIUk0AAHomAACAhAAA%2BgAAAIDoAAB1MAAA6mAAADqYAAAXcJy6UTwAAAJwUExURQAAAOvCc%2BWyUuWyT%2Be1VOm7Y%2Be%2Fcue2V%2FHEb%2BGuSuSyUue6ZOi4W%2Bm5Xei2Vee0UOe2VtmgMOm5Xei3V%2Be2Vue6Y9aeMeKsQeWxS92nP%2BWvRuWrOeaxSOu7YNqiNOasOuCrQunFfua2WeWyUOe1VdyoQuatPN%2BrRNupRd2pQuKsQuCtSMyWK8%2BVJOKpOeStQsGLIMSKF9ukN%2BqzRbF9F7qADeGpOeexRX5fM5ZsJcaYTLmSXlpFL21UOHleOotkII5pOpt4UraSawAAABoUDQsIBjotIHpdOY9oK6NyFaR%2FUolmQXdXM3JWNwIBAQAAABENCUg4JTgsIEw5IZNtNYllOnpYNIhoRYdrSwAAAAAAACYdFHVbPqB%2BWX9eOmxTNue7Zee5X96oPuO0WeWyT%2Be0U%2Be3Wei7ZNujN%2BOuR%2Be2V%2Bi7Zei6Yee0Uue6Y%2Be7Y%2BKwTue9bOe2WOi0UOeySuSrPeauQNujNuCtSOCqQuavReKqO%2BGnNuatPearOOesN9GWItOZJdacLOGoOMmWMM2ZMueuP%2BmuO%2BivPuW4YOi8adilQseMGMqPHNCWJd2lN8OSMNOdNemwQemxROe2VtilQNynQNGcM8ONJMaQKMmPHeSqOd2mOeOsPuWvRuKrQNymOsiNGcuQHs2VKM2WKsiULcaPIuGqPeazUeKuRtSeNMORL76EEMCFE8WNHrqDGb%2BIHNCXKN%2BnOeWtPuWsOs6aM8mWMbSFLrmAEb%2BGFqp2EaZ0E8ePH%2BSqOOesOOesOeKpOtOfPat5HLF6DahzDMGIGeWrOOqxQNajRLSIQb6FFeKpOdWmUquDSLKJS%2F%2F%2F%2F1QhmtkAAABgdFJOUwAEK33X1HkmAlfh%2FPzbkD8MefK%2FYgd51hp5%2FtYbDZLWGwIpdsX21hsYyNcbHtnYHB3Z2Bwf2tgcL%2BblLx%2Bwzdz4tyIEHzh28vn697xZEQIKI2iBpvr0t1UPAxZKy8tdEFDhfOQAAAABYktHRM%2BD3sJpAAAAB3RJTUUH6QkRCQQ0UiO4XwAAAQBJREFUGNNjYAACRiZmFlY2dg4GGODk4k5I5OHl4xeACggmJaekpiWkCwmLiIIFxDIys7JzcrPy8sUlwAKSBYV5RcUlUqVl0jIgvqxceUVlVXVNbV29vAKDopKyimpDY1NzS2tbe4eaOoOGZmdXd09vX%2F%2BEiZMmT9HSZtDRnTpt%2BoyZs2bPmTtv%2FgI9fQYDw4WLFi9Zumz5ipWlq1YbGTOYmK5Zu279ho2bNm%2Bp3brNzJzBwnL7jp27du%2FZu29%2Fx4GDVtYMNrZ29ocOHzl67HjbiZMOjk4Mzi6ubu4enqdOnznr5e3jC3SWn39AYFBwyLnQsPAImG8jo6JjYuPiwWwAjKZWgqTCB7sAAAAldEVYdGRhdGU6Y3JlYXRlADIwMjUtMDktMTdUMDk6MDQ6NTIrMDA6MDCyCIoGAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDI1LTA5LTE3VDA5OjA0OjUyKzAwOjAww1UyugAAACh0RVh0ZGF0ZTp0aW1lc3RhbXAAMjAyNS0wOS0xN1QwOTowNDo1MiswMDowMJRAE2UAAAAASUVORK5CYII%3D
+[version]: https://crates.io/crates/veecle-os
+[docs-badge]: https://img.shields.io/badge/docs.rs-veecle--os-teal?style=flat-square&logo=docs.rs
+[docs]: https://docs.rs/veecle-os
+[msrv-badge]: https://img.shields.io/crates/msrv/veecle-os?style=flat-square&logo=rust
```

Comment:
Just realized images link to themselves if they're not linked. For this I guess we could link to `rust-toolchain.toml`?